xls/modules/zstd/frame_header_test.cc

```diff
@@ -0,0 +1,382 @@
+// Copyright 2024 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define ZSTD_STATIC_LINKING_ONLY 1
+#include <zstd.h>
+
+#include <algorithm>
+#include <climits>
+#include <cstdint>
+#include <cstdio>
+#include <filesystem>
+#include <ios>
+#include <iostream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "absl/container/flat_hash_map.h"
+#include "absl/types/span.h"
+#include "fuzztest/fuzztest.h"
+#include "fuzztest/googletest_fixture_adapter.h"
+#include "gtest/gtest.h"
+#include "lib/common/zstd_errors.h"
```

Comment:
In other places where we include files that are from external projects and that have a very generic prefix (such as lib/ here), we typically then add the full name including the 'external' part to it, so that the project name is there for the developer to better see where things come from:

`external/zstd/lib/common/zstd_errors`

Examples where we use this pattern would be the [language server headers, that start with `common/`](https://github.com/google/xls/blob/e74fd95e1f8254b576cb6db2f1c9a99a2740f16c/xls/dslx/lsp/language_server_adapter.cc#L30-L32) or [z3 that otherwise would just start with `src/`](https://github.com/google/xls/blob/e74fd95e1f8254b576cb6db2f1c9a99a2740f16c/xls/solvers/z3_lec.cc#L51)

Reply:
Thanks for pointing that out! It's fixed now

Reply:
ah, I wasn't aware of this pattern!